src/Endatix.WebHost/appsettings.json

```diff
@@ -42,7 +42,7 @@
       "WelcomeEmailTemplateName": "welcome message"
     },
     "SendGridSettings": {
-      "ApiKey": "",
+      "ApiKey": "tqwieuqwte2735",

```

Comment:
This should be empty, correct? Or maybe it's intentional. Not sure if we should keep it. Better to remove it unless I am missing something

Reply:
I am getting some weird error, so I added this gibberish key to resolve it:

```
[ 02:23:47.167 Level:ERR] An unhandled exception has occurred while executing the request.
Microsoft.Extensions.Options.OptionsValidationException: DataAnnotation validation failed for 'SendGridSettings' members: 'ApiKey' with the error: 'The ApiKey field is required.'.
```

Reply:
This is data validation of the `SendGridSettings` - we should discuss this. For now please change it as it is misleading and offensive. It looks like a valid key, but it's not. We should rather:
1. Fix the issue with documentation and code, so devs don't have this experience
1. Add valid API Key for SendGrid - 💣 🔥 🦕  - No! No! No!
1. Another option - at least replace it with placeholder, which is honest -e.g. "{YOUR_SENDGRID_API_KEY_HERE}"

Option 2 is soo wrong. I vote for option 3. If you agree, please do it

Reply:
Option 3 is done. Thanks!